helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml

```diff
@@ -64,6 +64,7 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=$(ADDRESS)"
+            - "--timeout=180s"
```

Comment:
Attacher timeout needs to be configurable - which is the same change that you have done on the PR https://github.com/hpe-storage/co-deployments/pull/291/files#diff-142e27b15c7e115293526ed80db3eb1e3efac661324f402b178ab18b511fa822R67